src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+# Managing Mozilla.org
+
+Since the migration from .lang to Fluent, the L10n PM’s job is a lot simpler. The PM’s primary focus is on reviewing the string extractions provided by the bedrock team, provides feedback, manages the project through Pontoon and communicates important information to the communities.
+
+## Reviewing the PR
+
+When a PR is ready for localization review, through a script, the PR is pushed to the [www-l10n](https://github.com/mozilla-l10n/www-l10n/tree/master/) repository. The naming convention of the string ID is the file name followed by the first few words of the string, up to 7 words.
+
+### Reviewing new file
+
+* `en` is the source of a localizable file; en-US is a locale. Review is conducted in `en`.
+* Check that the demo URL works so you can review the text against the page on staging.
+* Check that all the brand and product names are in the form of placeable. If not, make sure that the term exists or it needs to be added.
```

Comment:
What's a placeable? Provide examples.

P.S. they're called "terms" in Fluent, let's stick to that.